### DISCUSSION TITLE
Support for Markdown and folders

### DISCUSSION THREAD

**Discussion by NetDevAutomate** [Quansight/ragna#209] [Ideas]
With support for Markdown and for folders, this would be an incredible tool to 'Chat' with Obsidian vaults.

A plugin for Obsian would I suspect of a lot of interest to the Obsidian community.

There are other Obsidian plugins for this but none as nice as Ragna, not even close.

**Comment by pmeier** [comment]
> With support for Markdown and for folders

These are two separate feature requests. I've opened #210 and #211 for you. The former is very simple to do in case you want to fix it yourself.

> A plugin for Obsian would I suspect of a lot of interest to the Obsidian community.

I'm not familiar with Obsidian so I don't know what "plugin" means in this context. In any case, this is something the community would need to take up, while Ragna certainly can be the backbone if the two issues above are addressed. Happy to answer questions in case some details are needed to create such a plugin.

**Reply by NetDevAutomate** [reply]
Fantastic!!! Thank you so much

**Comment by NetDevAutomate** [comment]
Closed as feature requests created.
